index.html

```diff
@@ -19,51 +19,259 @@
+       
+      </div>
+     
     </header>
     <main>
-    <section class="welcome-section">
-      <div class="headline">
-        <h1>Laura Jong</h1>
-        <h2>Software Developer</h2>
-        <p>
-          I can help you build a product , feature or website Look through some
-          of my work and experience! If you like what you see and have a project
-          you need coded, don’t hestiate to contact me.
-        </p>
+      <section class="welcome-section">
+        <div class="headline">
+          <h1>Kanombola Kano</h1>
+          <h2>Software Developer</h2>
+          <p>
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
+        </div>
+        <hr />
+        <section class="skills-container">
+          <ul class="box">
+            <li class="titled-ld">languages</li>
+            <li>Javascript</li>
+            <li>Ruby</li>
+            <li>html</li>
+            <li>CSS</li>
+          </ul>
+          <ul class="box">
+            <li class="titled-ld"><h4>Skills</h4></li>
+            <li>Github</li>
+            <li>Codepen</li>
+            <li>Terminal</li>
+            <li>Gitlab</li>
+          </ul>
+          <ul class="box">
+            <li class="titled-ld">Frameworks</li>
+            <li>Bootstrap</li>
+            <li>Selenium</li>
+            <li>Language</li>
+            <li>Language</li>
+          </ul>
+        </section>
+        <hr />
+        <div class="next-page-nav">
+          <div class="next-page right">
+            <a href="#cards"
+              ><img
+                src="./assets/arrow-svg.svg"
+                alt="Link to next section of page"
+            /></a>
+            <h5>Work</h5>
+          </div>
+          <ul class="social-media">
+            <li>
+              <a href="#"><i class="fa fa-github" aria-hidden="true"></i></a>
+            </li>
+            <li>
+              <a href="#"><i class="fa fa-linkedin" aria-hidden="true"></i></a>
+            </li>
+            <li>
+              <a href="#"><i class="fa fa-angellist" aria-hidden="true"></i></a>
+            </li>
+            <li>
+              <a href="#"><i class="fa fa-twitter" aria-hidden="true"></i></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img
+                  src="./assets/Icon-Medium.svg"
+                  alt="icon that links to medium"
+              /></a>
+            </li>
+          </ul>
+
+          <div class="next-page left">
+            <a href="#cards"
+              ><img
+                src="./assets/arrow-svg.svg"
+                alt="Link to next section of page"
+            /></a>
+            <h5>Work</h5>
+          </div>
+        </div>
+      </section>
+      <section id="cards">
+        <h3>MY RECENT WORK</h3>
+          <hr />
```

Comment:
- [ ] Kindly make sure this heading `<h3>` has the same styling in the mobile desktop version as the template one. check here 👇   the difference. 

Your design | the template
-----|-----
![image](https://user-images.githubusercontent.com/65068771/150423932-c616c017-31a9-4f9e-882d-d2d43b8f9dc7.png) | ![image](https://user-images.githubusercontent.com/65068771/150424173-2b0cdb06-f6c5-48ff-ba51-0fb27febdaa4.png)